content/geometry/object-vehicle/vehicle-structure.adoc

```diff
@@ -1088,11 +1113,16 @@ NOTE: Add a figure on an exemplary rotating warning light on the roof the vehicl
 |===
 
 
-== Grp_Mirror_Blindspot_Mounting_<part_idx>_<blindspot_mirror_mounting_idx>
+== Grp_Mirror_Blindspot_Mounting_<blindspot_mirror_mounting_idx>
+
+This group contains all parts of the vehicle's blindspot mirror mounting.
+It shall be a child of `<Grp_Exterior_Dynamic>` if mounted directly to the vehicle body, or a child of `<Grp_Door_Left>` or `<Grp_Door_Right>` if mounted to the door.
```

Comment:
The modal verb shall is used inconsistently. For example, section Grp_Mirror_Blindspot_Joint_Left uses the following phrase in the same contex:

"It is a child of the corresponding mirror mounting."

For consistency and readability reasons, we should use consistent phrases. We can either remove the shall or, if we we want to emphasize that there are conditions that require different handling, rephrase to:

"If mounted to the vehicle body directly, it is a child node of the group `<Grp_Exterior_Dynamic>`. If mounted to the door it is a child node of `<Grp_Door_Left>` or `<Grp_Door_Right>`."

Reply:
Yes, let's remove the shall here.

Reply:
Removed "shall" in this context across file